src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
+	private static CreditCardValidator validator = new CreditCardValidator();
+	private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	private static CreditCardVendor vendorMasterCard = vendors.get(1);
+	
+
+	@Test
+	public void testLengthOfNumberWhenCorrect() {
+		String number = "5584239583699571";
```

Comment:
This Credit Card number is duplicated in several tests (correct cases).
Define a constant instead of duplicating or follow advice above with creating test over several cases (including corner cases) with @ParameterizedTest